src/js/structs/DSLFilters.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * This class maintains a dictionary with all the currently available
+ */
+class DSLFilters {
```

Comment:
Could we call this `DSLFilterList` as we do other structs. This makes it easier to differentiate between the two